scripts/x-data-grid.exports.json

```diff
@@ -557,7 +557,6 @@
   { "name": "useGridSelector", "kind": "Variable" },
   { "name": "useGridVirtualization", "kind": "Function" },
   { "name": "useOnMount", "kind": "Function" },
-  { "name": "useResizeObserver", "kind": "Function" },
```

Comment:
Was this export intented?
It was imported from `@mui/x-data-grid/internals` in the pro package so I guess it's was a mistake.
If it's intended, I can re-export it.
If it's not intended, I can re-export it and deprecate it so that we can remove it in the next major.

@cherniavskii

Reply:
I think it should be fine to remove it from the public exports 👍

Reply:
Thanks :pray: 
I'll merge this PR once 7.8.0 is out :+1: